app/src/main/AndroidManifest.xml

```diff
@@ -3,6 +3,7 @@
 
     <!-- Application requires POST_NOTIFICATIONS only for ExoPlayer service -->
     <uses-permission android:name="android.permission.POST_NOTIFICATIONS" />
+    <uses-permission android:name="android.permission.INTERNET" />
```

Comment:
FYI, you don't really need the internet permission because it comes with srtdroid